jdaviz/conftest.py

```diff
@@ -154,6 +154,15 @@ def spectrum1d():
     return Spectrum1D(spectral_axis=spec_axis, flux=flux, uncertainty=uncertainty, meta=meta)
 
 
+@pytest.fixture
+def spectrum1d_nm():
```

Comment:
Seems like almost identical to `spectrum1d`. Must your test be in nm? If so, maybe do something like what cubeviz is doing with `_create_spectrum1d_cube_with_fluxunit` in `jdaviz/conftest.py`?

Reply:
It is, I was thinking of parameterizing the fixture itself but that would lead to all tests that use it being run with both the `AA` and `nm` versions (if I read the docs correctly). That cube one does look like the right way to do this, I'll consolidate.